src/pages/messaging/agent-chat-api/v3.7/data-structures/index.mdx

```diff
@@ -447,11 +447,11 @@ Apart from [Events](#events) and [Users](#users), there are also other common da
 | `properties` | optional  |      |
 | `access`     | optional  |      |
 
-## Chat summaries
+## Chat info
 
-**Chat summary** is similar to the **Chat** data structure. The difference is that **Chat** contains a `thread` object, while **Chat summary** includes `last_thread_summary` and `last_event_per_type`.
+**Chat info** is similar to the **Chat** data structure. The difference is that **Chat** contains a `thread` object, while **Chat info** includes `last_thread_info` and `last_event_per_type`.
 
-<CodeResponse version="v3.7" type="agent" title={'Sample Chat summary data structure'} json="chatSummary"/>
+<CodeResponse version="v3.7" type="agent" title={'Sample Chat info data structure'} json="chatInfo"/>
```

Comment:
for the code responses to show up you also have to rename the files under `/livechat-public-docs/payloads` - in this case its path is `/livechat-public-docs/payloads/messaging/v3.7/agent-chat-api/other-structures/chatSummary.json`